src/omnibar/omnibar.ts

```diff
@@ -578,7 +623,7 @@ function messageHandler(event: MessageEvent): void {
       BBOmnibarUserActivity.userRenewedSession();
       break;
     case 'environment-update':
-      handleEnvironmentUpdate(message.name);
+      handleEnvironmentUpdate(message.name, message.additionalInfo, message.url);
```

Comment:
I would prefer `description` over `additionalInfo` for the property name.

Reply:
I prefer `description` as well. Thanks!